lib/jekyll-import/importers/rss_podcast.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require_relative "rss"
+
+module JekyllImport
+  module Importers
+    class RSSPodcast < RSS
+      # Process the import.
+      #
+      # source - a URL or a local file String.
+      #
+      # Returns nothing.
+      def self.process(options)
+        source = options.fetch("source")
+        frontmatter = options.fetch("frontmatter", [])
+        body = options.fetch("body", ["description"])
+        overwrite = options.fetch("overwrite", true)
+
+        content = ""
+        uri = URI.parse(source)
+        uri.open { |s| content = s.read }
```

Comment:
Any reason you opted for `URI.parse(source).open` instead of `open(source)` used in `rss.rb`?

Reply:
CI failed when I used the “open” call.
Something to do with Kernel#open being insecure.